docs/en/observability/infrastructure-threshold-alert.asciidoc

```diff
@@ -37,4 +45,40 @@ image::images/alert-preview.png[Preview alerts]
 You can extend your alerts by connecting them to actions that use the following supported built-in integrations.
 
 [role="screenshot"]
-image::images/action-type-metrics.png[Anomalies chart]
+image::images/action-type-metrics.png[Action types]
+
+[float]
+=== Action variables
+
+This section details the variables that infrastructure threshold alerts will send to your actions.
+
+[float]
+==== Basic variables
+
+[role="screenshot"]
+image::images/basic-variables.png[The default infrastructure threshold alert message detailing basic variables]
+
+The default message for a infrastructure threshold alert shows an example of how to use its most basic variables.
+
+context.group:: This variable resolves to the **group** on which the alert's condition was detected. For Inventory alerts, this will be the name of a monitored host, pod, container, etc. For Metric Threshold alerts, this will be the value of the field specified in the **Create alert per** field, or `*` if the alert is configured to aggregate your entire infrastructure.
+context.alertState:: This variable will resolve to `ALERT`, `NO DATA`, or `ERROR`, depending on why this action is firing. `ALERT` means the alert's condition has been detected, `NO DATA` means no data was returned for the time period that the alert queried, and `ERROR` indicates an error when trying to query the data.
+context.reason:: This variable provides a plain English description of why the alert is in its current state. For each of the alert's conditions, it will include the **detected value** of the monitored **metric**, and a description of the **threshold** it crossed.
+context.timestamp:: This variable resolves to the timestamp at which the alert evaluated.
+
+
+[float]
+==== Advanced variables
+
+[role="screenshot"]
+image::images/advanced-variables.png[The default infrastructure threshold alert message detailing advanced variables]
+
+There may be cases when you'd like to customize your action message instead of using `context.reason` to provide all the information you need. Infrastructure threshold alerts will also provide some advanced context variables that have a tree structure.
```

Comment:
```suggestion
Instead of using `context.reason` to provide all the information you need, there may be cases when you'd like to customize your action message. Infrastructure threshold alerts provide advanced context variables that have a tree structure.
```